tests/main/set-proxy-store/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Check that the setting proxy.store config works
+environment:
+    SNAP_NAME: test-snapd-tools
+    SNAP_VERSION_PATTERN: \d+\.\d+\+fake1
+    BLOB_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
```

Comment:
you could do systems: [-ubuntu-core-16-*] instead of exit on all the stages

Reply:
Pushed this to https://github.com/snapcore/snapd/pull/4203.